alf/utils/lean_function.py

```diff
@@ -301,3 +301,56 @@ def _infer_device_type(*args):
         return "cuda"
     else:
         return device_types[0]
+
+
+class SplitBatchRunner(torch.nn.Module):
+    """Split the input into smaller batches and run the model on each batch.
```

Comment:
Need to clarify that lean_function will be used.

Reply:
comment added